pepper-apis/pom.xml

```diff
@@ -25,6 +25,7 @@
         <elasticsearch.version>7.17.10</elasticsearch.version>
         <velocity.version>2.3</velocity.version>
         <jackson.version>2.12.1</jackson.version>
+        <redisson.version>3.24.2</redisson.version>
```

Comment:
Extracting this so we only have to update the version in one place in the future.